test/runtime/samples/store-auto-resubscribe-immediate/_config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	html: `{"answer":4}`
```

Comment:
can you rebase against master and make sure `npm run lint` passes? we just updated our eslint config to allow only single quotes